pipework

```diff
@@ -52,15 +52,15 @@ installed () {
 # First step: determine type of first argument (bridge, physical interface...),
 # Unless "--wait" is set (then skip the whole section)
 if [ -z "$WAIT" ]; then 
-  if [ -d /sys/class/net/$IFNAME ]
+  if [ -d "/sys/class/net/$IFNAME" ]
   then
-    if [ -d /sys/class/net/$IFNAME/bridge ]; then
+    if [ -d "/sys/class/net/$IFNAME/bridge" ]; then
       IFTYPE=bridge
       BRTYPE=linux
-    elif installed ovs-vsctl && ovs-vsctl list-br|grep -q ^$IFNAME$; then
+    elif installed ovs-vsctl && ovs-vsctl list-br|grep -q "^${IFNAME}$"; then
       IFTYPE=bridge
       BRTYPE=openvswitch
-    elif [ $(cat /sys/class/net/$IFNAME/type) -eq 32 ]; then # Infiniband IPoIB interface type 32
+    elif [ "$(cat "/sys/class/net/$IFNAME/type")" -eq 32 ]; then # Infiniband IPoIB interface type 32
```

Comment:
Putting quotes around the `$()` could change the behavior of the test. Before, whitespace around the number in the file was being ignored. Now it's part of the test.

I expect the file doesn't contain any whitespace, so it may not matter.